src/customize-pop/application-notifications.md

```diff
@@ -0,0 +1,31 @@
+# Application Notifications
+
+Settings to control application notification behavior are located in the `Settings` ➞ `Notifications` tab.
+
+![Notifications Settings](/images/application-settings/notifications-settings.png)
+
+| Option | Function |
+|--------|----------|
+| Do Not Disturb | Disable all application notification pop-ups globally. Notifications will still queue up in the notiications drop-down while this setting is enabled. |
+| Lock Screen Notifications | Enable setting toggling for the `Lock Screen Notifications` and `Show Message Content on Lock Screen` options. |
+
+## Viewing Notifications
+
+Application notifications appear in the upper center of the screen. Click the date and time to view recent notifications. You can also globally disable notifications by toggling `Do Not Disturb`.
+
+![Recent Notifications](/images/application-settings/recent-notifications.png)
+
+## Configuring an Application's Notification Behavior
+
+Select an application from the list to configure its notification behavior.
+
+![Single App Options](/images/application-settings/single-app-options.png)
+
+| Option | Function |
+|--------|----------|
+| Notifications | Toggle an application's notifications on or off. |
+| Sound Alerts | Toggle sound alerts that will accompany notifications for this application. |
+| Notification Popups | Enable notification popups on the desktop; notifications will continue to appear in the notification list. |
+| Show Message Content in Popups | Include the notification's written content in the popup. |
+| Lock Screen Notifications | Enable additional sub-menu options for lock screen notifications. These options are set on an app by app basis. |
```

Comment:
> Enable additional sub-menu options for lock screen notifications. These options are set on an app by app basis.

You're already at the app by app basis, and there is no sub-menu beyond the one you're currently talking about. I would reword/simplify this. It could be something like:

> Enable or disable showing this specific application's notifications on the lock screen.

Reply:
Added your suggested phrasing